subscribe_test.go

```diff
@@ -259,6 +259,44 @@ func TestSubscribe(t *testing.T) {
 	testSubscribe(t, 3)
 }
 
+func TestSubscribeWithDebug(t *testing.T) {
```

Comment:
You should assert that the payload has been logged using https://pkg.go.dev/go.uber.org/zap@v1.24.0/zaptest/observer

Reply:
@dunglas I added a test to check payload field is present in the expected log entry but I can't figure out how to pass and test the content of the payload.

Reply:
You need to create a JWT containing the data your expect: https://mercure.rocks/spec#payload